tracing-appender/src/rolling.rs

```diff
@@ -629,7 +586,12 @@ impl io::Write for RollingWriter<'_> {
     fn write(&mut self, buf: &[u8]) -> io::Result<usize> {
         let written = (&*self.inner).write(buf)?;
         self.current_size.fetch_add(
-            u64::try_from(written).expect("usize to u64 conversion"),
+            u64::try_from(written).map_err(|e| {
+                io::Error::new(
+                    io::ErrorKind::Other,
+                    format!("Failed to convert usize to u64: {}", e),
+                )
+            })?,
```

Comment:
note: I think it’s very fine to keep `expect` at this place as I don’t expect this conversion to fail

Reply:
True, because usize will never be an invalid u64 right? It would panic earlier. Mmh I think I like it better with the expect because it's more explicit that it shouldn't really fail there.

Reply:
Agreed.